nautobot/utilities/tests/test_forms.py

```diff
@@ -615,3 +653,230 @@ def test_api_select_add_query_param_with_utf8(self):
+                    ("id", "Id"),
+                    ("last_updated", "Last updated"),
+                    ("latitude", "Latitude"),
+                    ("locations", "Locations within this Site (slugs or IDs)"),
+                    ("longitude", "Longitude"),
+                    ("name", "Name"),
+                    ("physical_address", "Physical address"),
+                    ("power_panels", "Power panels (name or ID)"),
+                    ("prefixes", "Prefixes"),
+                    ("q", "Search"),
+                    ("rack_groups", "Rack groups (slug or ID)"),
+                    ("racks", "Racks"),
+                    ("region", "Region (slug)"),
+                    ("region_id", "Region (ID)"),
+                    ("shipping_address", "Shipping address"),
+                    ("slug", "Slug"),
+                    ("status", "Status"),
+                    ("tag", "Tags slug"),
+                    ("tenant", "Tenant (slug or ID)"),
+                    ("tenant_group", "Tenant Group (slug)"),
+                    ("tenant_group_id", "Tenant Group (ID)"),
+                    ("tenant_id", 'Tenant (ID) (deprecated, use "tenant" filter instead)'),
+                    ("time_zone", "Time zone"),
+                    ("vlan_groups", "Vlan groups (slug or ID)"),
+                    ("vlans", "Vlans"),
+                ],
+            )
+
+        # TODO(Culver): Assertion doesn't like instantiated filters
+        # with self.subTest("Assert form generates the correct base_filters"):
+        #     self.assertEqual(form.filterset_filters, StatusFilterSet().filters)
+        #     self.assertEqual(site_form.filterset_filters, SiteFilterSet().filters)
+
+        with self.subTest("Assert lookup_field, lookup_value & lookup_type fields has accurate attributes"):
+            self.assertEqual(
+                form.fields["lookup_field"]._choices,
+                [
+                    (None, "---------"),
+                    ("color", "Color"),
+                    ("content_types", "Content type(s)"),
+                    ("created", "Created"),
+                    ("id", "Id"),
+                    ("last_updated", "Last updated"),
+                    ("name", "Name"),
+                    ("q", "Search"),
+                    ("slug", "Slug"),
+                ],
+            )
+            self.assertEqual(
+                form.fields["lookup_field"].widget.attrs,
+                {"class": "nautobot-select2-static lookup_field-select", "placeholder": "Field"},
+            )
+
+            self.assertEqual(
+                form.fields["lookup_type"].widget.attrs,
+                {
+                    "class": "nautobot-select2-api lookup_type-select",
+                    "placeholder": None,
+                    "data-query-param-field_name": '["$lookup_field"]',
+                    "data-contenttype": "extras.status",
+                    "data-url": "/api/lookup-choices/",
```

Comment:
Should probably use `reverse()` here and below rather than hard-coding this URL.